src/delete.js

```diff
@@ -0,0 +1,51 @@
+const todolist = JSON.parse(localStorage.getItem('list')) || [];
+export const forms = document.querySelector('.todo-form');
+export const input = document.querySelector('.todo-input');
+export const clear = document.querySelector('.submit');
+
+const deletefun = () => {
+  if (todolist[0] === null) return;
+  const edit = document.querySelectorAll('.place-input');
+  edit.forEach((trigger) => {
+    trigger.addEventListener('keyup', (e) => {
+      trigger.removeAttribute('readonly');
+      if (e.key === 'Enter') {
+        todolist[trigger.id].description = trigger.value;
+        localStorage.setItem('list', JSON.stringify(todolist));
+        window.location.reload();
+      }
+    });
+  });
+  const checker = document.querySelectorAll('.box');
+  checker.forEach((boox) => {
+    boox.addEventListener('change', (e) => {
+      if (e.target.checked) {
+        todolist[boox.id].completed = true;
+        boox.parentElement.lastElementChild.classList.add('checked');
+      } else if (!e.target.checked) {
+        todolist[boox.id].completed = false;
+        boox.parentElement.lastElementChild.classList.remove('checked');
+      }
+      localStorage.setItem('list', JSON.stringify(todolist));
+    });
+  });
+
+  const singledelete = document.querySelectorAll('.del');
+  const itemz = [...singledelete];
+  itemz.forEach((deletebtn) => {
+    deletebtn.addEventListener('click', () => {
+      todolist[deletebtn.id].completed = true;
+      const onedelete = todolist.filter((todo) => todo.completed === false);
+      localStorage.setItem('list', JSON.stringify(onedelete));
+      window.location.reload();
+    });
```

Comment:
Please make sure we delete only one row if we selected many tasks and delete Just one , It should not be removing the other checked task.
![DeleteMax](https://user-images.githubusercontent.com/66979348/193591872-759e2e5a-eadf-4fcd-9ec6-c2dd01194267.JPG)
![DeleteMax2](https://user-images.githubusercontent.com/66979348/193591876-7f396d38-d9c9-4857-b72e-2fd6b05a2e4a.JPG)
![ReplaceChild](https://user-images.githubusercontent.com/66979348/193591879-cca1dfd1-6619-4eed-803e-2cd820a975b2.JPG)